codes/classical/properties/covering.yml

```diff
@@ -0,0 +1,46 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: covering
+
+name: 'Covering code'
+
+description: A code \(C\) is called \(\rho\)-covering code if \(\forall \bm{v} \in \mathbb{F}_q^{n}\) \(\exists \bm{c} \in C\) such that \(d(\bm{c},\bm{v})\leq \rho\). The covering radius of a code \(C\), \(\rho(C)\) is smallest \(\rho \in \mathbb{Z}^{+}\) such that \(C\) is \(\rho\)-covering, i.e.
+  \begin{align}
+    \rho(C)=\max_{{\bm{v}\in \mathbb{F}_q^{n}}}\min_{{\bm{c}\in C}}d(\bm{v},\bm{c})
+  \end{align}
+features:
+    rate: ''
+
+realizations:
+  - 'Data compression both with or without compression'
+  - 'Football-pool problem: smallest number of bets(on n matches) needed to guarantee at least one bet has at most \(\rho\) errors'
+
+notes:
+    - 'A code \(C\) with \(d(C)=d\) satisfies
+    \begin{align}
+      \rho(C)\geq \frac{|d-1|}{2} \label{perfect-ref}
+    \end{align}'
+    - '\textbf{(Redundancy Bound)}A code \(C\) with \([n,k]\) satisfies
+    \begin{align}
+      \rho(C)\leq n-k
+    \end{align}'
+    - '\textbf{(Sphere Covering Bound)} A code \(C\)  over \(\mathbb{F}_q^{n}\) satisfies
+    \begin{align}
+        \rho(C)\leq \min{\left(p~\bigg\rvert \sum_{i=0}^{p} {n \choose i}(q-1)^{i}|C| \geq q^{n}\right)} \label{spherepacking-perfect-label}
+    \end{align}'
+    - 'Covering codes can be nonlinear'
+    - 'In general finding covering radius for a given code is difficult. Complexity analysis as well as an extensive study on bounds can be found in \cite{manual:{G. Cohen, I. Honkala, S. Litsyn, A. Lobstein, Covering Codes, Elsevier (1997).}}.'
+    - 'Perfect codes satisfy \eqref{perfect-ref} and \eqref{spherepacking-perfect-label} with equality. Converse is also true.'
```

Comment:
Cousin entry for perfect with this as detail

Reply:
I think covering should be parent. that is why I put covering as parent to perfect in perfect.yml. there seems to be no child relation defined? so I put this information in notes.